test/Spec.hs

```diff
@@ -0,0 +1,94 @@
+import Reader
+import Hedgehog
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import Test.Tasty
+import Test.Tasty.HUnit hiding (assert)
+import Test.Tasty.Hedgehog
+
+genString :: Gen String
+genString = Gen.string (Range.linear 0 100) Gen.unicode
+
+genMultiLineString :: Gen String
+genMultiLineString = do
+    liness <- Gen.list (Range.linear 1 100) (Gen.string (Range.linear 0 50) Gen.unicode)
+    return (unlines liness)
+
+prop_wordCount :: Property
+prop_wordCount = property $ do
+    content <- forAll genString
+    let wordsContent = words content
+    resultWordCount <- evalIO $ wordCounter content
+    assert (resultWordCount == length wordsContent)
+
```

Comment:
This test duplicates the implementation of the function. What is it supposed to test?